exampleservice/CustomerService/Handler/CheckSessionHandler.cs

```diff
@@ -0,0 +1,63 @@
+﻿using exampleservice.CustomerService.Contract;
+using exampleservice.CustomerService.Controller;
+using exampleservice.CustomerService.Events;
+using exampleservice.Framework.Abstract;
+using exampleservice.Framework.BaseFramework;
+using System;
+using System.Threading.Tasks;
+
+namespace exampleservice.CustomerService.Handler
+{
+    internal class CheckSessionHandler : CustomerHandlerBase<CheckSessionCommand>
+    {
+        private IMessageBus bus;
+        private ICustomerServiceDataBaseRepository dataBaseRepository;
+
+        public CheckSessionHandler(IMessageBus bus, ICustomerServiceDataBaseRepository dataBaseRepository)
+        {
+            this.bus = bus ?? throw new ArgumentNullException(nameof(bus));
+            this.dataBaseRepository = dataBaseRepository ?? throw new ArgumentNullException(nameof(dataBaseRepository));
+        }
+
+        internal override async Task<EventBase> Handle(CheckSessionCommand command)
+        {
+            // 1. Session Valid?
```

Comment:
i don't get this comment, sorry

Reply:
this was one question we also asked via slack - this would probably be a part where we could add the (maybe) requried procedure steps, even if it wouldnt make much sense here

Reply:
We checked where we have different "steps" and if it makes sense in our case - so not useful here